app/assets/javascripts/jquery.persistentModal.js

```diff
@@ -105,20 +98,23 @@
       }
 
       function persistentModalHtml(modalBody) {
-        modalHtml = [
+        var modalHtml = [
+
           // the 'data-backdrop="static"' and 'data-keyboard="false"' attributes keep the modal from closing on clickaway or escape, respectively
           '<div class="modal persistent-modal" tabindex="-1" role="modal" data-persistent-modal-url="' + linkTarget + '" data-backdrop="static" data-keyboard="false">',
-          '  <div class="modal-dialog">',
-          '    <div class="modal-header">',
-          '      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>',
-          '      <h3 class="modal-title">TITLE</h3>',
-          '    </div>',
+          '  <div class="modal-dialog modal-lg">',
           '    <div class="modal-content">',
+          '      <div class="modal-header">',
+          '        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>',
```

Comment:
I'd like to leave this the way it is for styling readability, if possible. Also open to other suggestions.